pkg/cloudprovider/providers/openstack/openstack_volumes.go

```diff
@@ -236,7 +236,16 @@ func (volumes *VolumesV3) expandVolume(volumeID string, newSize int) error {
 	createOpts := volumeexpand.ExtendSizeOpts{
 		NewSize: newSize,
 	}
+
+	// save initial microversion
+	mv := volumes.blockstorage.Microversion
```

Comment:
just curious, why we can't have a client with latest version so that we can always call this kind of version specific feature?

Reply:
because Cinder can have an old version and if we permanently set the microversion, then all further operations using the same client will fail.

Reply:
yes, I fully understand the microversion concept
I mean, this kind of 3.42 hard coding why we not have a latest client object and 
use that object for *all* future features like this instead of hard code every where

Reply:
It is simpler to set the required microversion for an operation rather than set the latest one for all operations, because it can break stuff, which was available in lower microversions.

Reply:
@jichenjc @kayrus Even we have latest version, unless specific microversion is set , that feature is not enabled , In this case unless we set 3.42 microversion, live resize is not enabled

Reply:
thanks, I am not aware that, at this is different to what I learn from nova side.. thanks for clarify this